cli/pack/app_dir.go

```diff
@@ -336,6 +339,15 @@ func copyProjectFiles(dst string, ctx *context.Ctx) error {
 				return false, fmt.Errorf("Failed to get file rel path: %s", err)
 			}
 
+			// We need to ignore the run_dir, data_dir and log_dir folders because it
+			// contains the files of the running instance, which can then block application pack.
+			// More details: https://github.com/tarantool/cartridge-cli/issues/494
+			for _, path := range defaultLocalPaths {
```

Comment:
Why not just `for _, path := range project.GetDefaultLocalRunningPaths()` ?

Reply:
In this case we will call ``project.GetDefaultLocalRunningPaths()`` many times (versus only one call) - because this function called for every path in project